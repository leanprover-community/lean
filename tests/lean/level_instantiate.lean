#check level.instantiate

#eval level.to_string $ level.instantiate (level.param `x) [(`x, level.param `y)]

#eval level.to_string $ level.instantiate (level.param `x) [(`x, level.param `y), (`y, level.param `z)]

#eval level.to_string $ level.instantiate (level.param `x) [(`x, level.param `y), (`x, level.param `z)]

#eval level.to_string $ level.instantiate (level.max (level.param `x) (level.param `y)) [(`x, level.param `y), (`y, level.param `z)]
