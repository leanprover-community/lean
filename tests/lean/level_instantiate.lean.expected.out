level.instantiate : level → list (name × level) → level
"y"
"y"
"y"
"max y z"
